packages/menlo/data_stream/dlp/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,155 @@
+---
+description: Mapping of Menlo DLP Fields from API into ECS fields.
+processors:
+  - set:
+      field: ecs.version
+      value: 8.11.0
+  - rename:
+      field: message
+      target_field: event.original
+      ignore_missing: true
+  - set:
+      field: event.kind
+      value: event
+  - set:
+      field: "event.module"
+      value: "menlo"
+  - json:
+      field: event.original
+      tag: 'json_decoding'
+      target_field: json
+  - rename:
+      field: json.event.event_id
+      target_field: event.id
+      ignore_missing: true
+  - date:
+      field: "json.event.event_time"
+      formats:
+        - "ISO8601"
+      target_field: "event.created"
+      timezone: "UTC"
+      on_failure:
+        - append:
+            field: error.message
+            value: 'Processor {{{_ingest.on_failure_processor_type}}} with tag fail-{{{_ingest.on_failure_processor_tag}}} in pipeline {{{_ingest.pipeline}}} failed with message: {{{_ingest.on_failure_message}}}'
```

```suggestion
            value: 'Processor {{{_ingest.on_failure_processor_type}}} with tag {{{_ingest.on_failure_processor_tag}}} in pipeline {{{_ingest.pipeline}}} failed with message: {{{_ingest.on_failure_message}}}'
```
Same change for rest of the pipelines.

---

`tag` must be specified in the processor because of this message asking for tag.
https://github.com/elastic/integrations/blob/main/packages/ti_threatconnect/data_stream/indicator/elasticsearch/ingest_pipeline/default.yml#L94